src/shared/components/MultiFileSelect/MultiFileSelect.tsx

```diff
@@ -189,14 +198,14 @@ export const MultiFileSelect: React.FC<MultiFileSelectProps> = ({
         maxSize={step === 'video' ? maxVideoSize : maxImageSize}
         onUploadFile={handleUploadFile}
         onReAdjustThumbnail={handleReAdjustThumbnail}
-        isLoading={isLoading}
+        isLoading={isVideoLoading || isImgLoading}
         fileType={step}
         title={step === 'video' ? 'Select video file' : 'Add thumbnail image'}
         thumbnailUrl={files.thumbnail?.url}
         paragraph={
           step === 'video'
-            ? `Maximum 10GB. Accepts any format supported by your browser.`
-            : `Accepts any format supported by your browser.`
+            ? `Maximum 10GB. Prefered format is MP4 (H.264) or WEBM (VP8,VP9)`
+            : `Preffered 16:9 image ratio. Accepting JPEG or WEBP formats.`
```

Comment:
We transcode the image anyway so the input format doesn't really matter

```suggestion
            : `Preferred 16:9 image ratio`
```